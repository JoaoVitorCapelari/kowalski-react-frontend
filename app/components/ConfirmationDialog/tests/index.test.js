// import React from 'react';
// import { shallow } from 'enzyme';

// import ConfirmationDialog from '../index';

describe('<ConfirmationDialog />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
