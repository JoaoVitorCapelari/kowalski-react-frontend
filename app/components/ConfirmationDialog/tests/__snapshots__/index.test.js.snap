// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmationDialog /> renders 1`] = `"<div class=\\"modal is-active\\"><div class=\\"modal-background\\"></div><div class=\\"modal-card\\"><header class=\\"modal-card-head\\"><p class=\\"modal-card-title\\">Confirmation</p><button class=\\"delete\\" aria-label=\\"close\\"></button></header><section class=\\"modal-card-body\\">something</section><footer class=\\"modal-card-foot\\"><button class=\\"button is-success\\">Confirm</button><button class=\\"button\\">Cancel</button></footer></div></div>"`;
